discussions-store/discussions-store-api/src/main/java/org/xwiki/contrib/discussions/store/meta/DiscussionContextMetadata.java

```diff
@@ -145,6 +145,16 @@
      */
     public static final String DISCUSSIONS_NAME = "discussions";
 
+    /**
+     * Discussion context metadata key.
```

Comment:
I don't remember the conclusion we reached on using `@since` and `@Unstable` on contrib code.
An `@since` as a documentation could be interesting though.

Reply:
Right I missed it there